modules/sources/contracts/fetch-tokenpair-data.ts

```diff
@@ -214,6 +265,21 @@ function addEffectivePriceCallsToMulticaller(
     });
 }
 
+// call queryRemoveLiquiditySingleTokenExactIn from tokenA(BPT)->tokenB with 100USD worth of BPT
+function addBptEffectivePriceCallsToMulticaller(
```

Comment:
This is related to add/remove liquidity steps only, right? Please make sure those are present only in the PR with add/remove steps